Makefile

```diff
@@ -85,38 +85,24 @@ $(FC_TEST_DATA_PATH)/ltag:
 	GO111MODULE=off GOBIN=$(abspath $(FC_TEST_DATA_PATH)) \
 	go get github.com/kunalkushwaha/ltag
 
-tools/firecracker-builder-stamp: tools/docker/Dockerfile
-	docker build \
-		-t localhost/$(FIRECRACKER_BUILDER_NAME):$(DOCKER_IMAGE_TAG) \
-		-f tools/docker/Dockerfile \
-		tools/docker
-	touch $@
+$(FIRECRACKER_DIR):
+	- git clone https://github.com/firecracker-microvm/firecracker.git $(FIRECRACKER_DIR)
 
 .PHONY: test-images
 test-images: $(FIRECRACKER_BIN) $(JAILER_BIN)
 
-$(FIRECRACKER_BIN) $(JAILER_BIN): tools/firecracker-builder-stamp
-	mkdir -p build
-	docker run --rm -it \
-		--user $(UID):$(GID) \
-		--volume $(CURDIR)/build:/artifacts \
-		--volume $(CARGO_CACHE_VOLUME_NAME):/usr/local/cargo/registry \
-		-e HOME=/tmp \
-		--workdir=/firecracker \
-		localhost/$(FIRECRACKER_BUILDER_NAME):$(DOCKER_IMAGE_TAG) \
-		cargo build --release \
-		--target-dir=/artifacts --target $(FIRECRACKER_TARGET) \
-		-p firecracker -p jailer
-	cp build/$(FIRECRACKER_TARGET)/release/firecracker $(FIRECRACKER_BIN)
-	cp build/$(FIRECRACKER_TARGET)/release/jailer $(JAILER_BIN)
+$(FIRECRACKER_BIN) $(JAILER_BIN): $(FIRECRACKER_DIR)
+	# Strip currently commented out because it's being fixed in Firecracker
+	# and is under review.
+	$(FIRECRACKER_DIR)/tools/devtool -y build --release #&& \
```

Comment:
What we need to run the devtool script? Python 2 or 3 and Docker?

Reply:
I think they are using Docker. https://github.com/firecracker-microvm/firecracker/blob/391eb1afc2a66b1bdccac0ea083cf56b2ec55a4d/tools/devtool

Reply:
@kzys as @RoyceDavison pointed out, it's using Docker behind the scenes launched through a bash script.